index.html

```diff
@@ -2509,34 +2510,75 @@ <h3>
         </h3>
 
         <p>
-The <a>DID document</a> is serialized following the rules in the JSON processor,
-with one addition: <a>DID documents</a> MUST include the <code>@context</code>
-property.
+The <a>DID document</a> MUST be serialized as a JSON document,
+with one additional requirement: <br/> 
+The <a>DID document</a> MUST include the <code>@context</code> property.
```

Comment:
Since `@context` isn't a "property" of the DID subject, can we call it something else?

```suggestion
The JSON document MUST include the <code>@context</code> member.
```

Reply:
-1 to using the term "member".

both `id ` and `@context` are "properties of a did document"... only `id` is required.

http://json-schema.org/understanding-json-schema/reference/object.html#:~:text=%2C%20%22object%22%5D-,Properties,used%20to%20validate%20that%20property.

Reply:
Agree with @OR13 -- this whole contemplation of different classes of properties in a DID Document is new and does not have consensus.

@peacekeeper -- are you objecting to this PR if your suggested change is not made?

Reply:
`id` is a [property of the DID subject](https://w3c.github.io/did-core/#dfn-id). `@context` is not.

Reply:
`id` is also a `property` of a did document, in the sense that JSON objects have properties... perhaps the ADM does not have properties in your mind?

IMO the ADM also supports `@context` as a property, because otherwise it would not be possible to use the ADM to produce JSON-LD.